inc/consent/namespace.php

```diff
@@ -8,23 +8,43 @@
 namespace Altis\Privacy\Consent;
 
 use Altis;
+use Altis\Consent;
 
 /**
  * Kick it off.
  */
 function bootstrap() {
+	$config = Altis\get_config()['modules']['privacy']['consent'];
+
+	// Unless the consent module has been deactivated, load the plugins.
+	if ( empty( $config ) ) {
+		return;
+	}
+
 	add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_plugins', 1 );
+
+	// Filter native analytics consent cookie prefix.
+	add_filter( 'altis.analytics.consent_cookie_prefix', __NAMESPACE__ . '\\enable_analytics_consent' );
 }
 
 /**
  * Load plugins that are part of the consent module.
  */
 function load_plugins() {
-	$config = Altis\get_config()['modules']['privacy']['consent'];
+	require_once Altis\ROOT_DIR . '/vendor/altis/consent-api/plugin.php';
+	require_once Altis\ROOT_DIR . '/vendor/altis/consent/plugin.php';
+}
 
-	// Unless the consent module has been deactivated, load the plugins.
-	if ( $config ) {
-		require_once Altis\ROOT_DIR . '/vendor/altis/consent-api/plugin.php';
-		require_once Altis\ROOT_DIR . '/vendor/altis/consent/plugin.php';
+/**
+ * Filter the Analytics plugin consent cookie prefix.
+ *
+ * @param string|null $cookie_prefix The analytics consent cookie prefix or null.
+ * @return string|null
+ */
+function enable_analytics_consent( ?string $cookie_prefix ) {
```

Comment:
This function is a little obtuse. From the perspective of "enabling altis consent" it doesn't make a lot of sense how it's related to the cookie prefix -- those things seem like two different things from an outside perspective. I think this can be resolved by additional documentation explaining how passing a cookie prefix is related to enabling consent.

Reply:
I adapted the other PR based on this feedback so this integration is much simpler now.